jobs/build-arch.Jenkinsfile

```diff
@@ -324,7 +324,8 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
                 pipeutils.shwrapWithAWSBuildUploadCredentials("""
                 cosa buildupload --skip-builds-json s3 \
                     --aws-config-file \${AWS_BUILD_UPLOAD_CONFIG} \
-                    --acl=${acl} ${s3_stream_dir}/builds
+                    --acl=${acl} ${s3_stream_dir}/builds \
+                    --arch=${basearch}
```

Comment:
if it would help the "diff-ability" should we make this change in the build job too?